docs/cli-help.txt

```diff
@@ -13,27 +13,27 @@ cert. Major SUBCOMMANDS are:
   register             Perform tasks related to registering with the CA
   rollback             Rollback server configuration changes made during install
   config_changes       Show changes made to server config during installation
+  update_symlinks      Update cert symlinks based on renewal config file
   plugins              Display information about installed plugins
+  certificates         Display information about certs configured with Certbot
```

Comment:
Which is the one that we keep in sync with the current release because the docs pulls it straight from master then?